src/Common/src/Interop/Windows/mincore/Interop.Environment.cs

```diff
@@ -16,6 +16,10 @@ internal static unsafe partial class mincore
 
         [DllImport(Libraries.Kernel32, EntryPoint = "ExitProcess")]
         internal static extern void ExitProcess(int exitCode);
+
+        [MethodImpl(MethodImplOptions.InternalCall)]
```

Comment:
The motivation for the current scheme is in https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/interop-guidelines.md